frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunRow.tsx

```diff
@@ -32,6 +33,7 @@ const PLRStatus: React.FC<PLRStatusProps> = ({ obj }) => {
 };
 
 const PipelineRunRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj }) => {
+  const version = usePipelineOperatorVersion(obj.metadata.namespace)?.version;
```

Comment:
better to use minor and major property returns by this hook as version might be not a valid number as per the Javascript.

Reply:
Updated PR with the said changes, PTAL